health/health.go

```diff
@@ -159,33 +179,39 @@ func CheckStatus() map[string]string { // TODO(stevvooe) this needs a proper typ
 }
 
 // Register associates the checker with the provided name. We allow
-// overwrites to a specific check status.
-func Register(name string, check Checker) {
-	mutex.Lock()
-	defer mutex.Unlock()
-	_, ok := registeredChecks[name]
+// overwrites to a specific check status. registry may be nil to use the
+// default registry.
+func Register(registry *Registry, name string, check Checker) {
```

Comment:
I modeled this after `http.ListenAndServe`. It accepts a nil parameter for the handler to use the default mux. If this was a method on `Registry`, it would need to be used like

    health.DefaultRegistry.Register(...)

...which seems awkward to me - I would rather users never have to touch DefaultRegistry directly.